### ISSUE TITLE
ng_controller_name false positives

### ISSUE THREAD

**Issue by alexlrobertson** [Gillespie59/eslint-plugin-angular#158]
In a unit test, I'm using `element.controller()` to get an element’s controller, which is raising a false error.

```javascript
html = '<grid query-api="queryApi"></grid>';
element = $compile(html);
upGridController = element.controller('upGrid');
```
